export const handleError = (message) => {
    message ? 
        alert(message) :
        alert('Something went wrong!')
}